discord-bot/noxfile.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding=utf-8 -*-
+"""Automated linting, formatting, and typechecking."""
+import nox
+from nox.sessions import Session
+
+
```

Comment:
we use pre-commit .. please do not add other project-specific style enforcement tools!